internal/app/httpserver/shorten/delete_short_urls.go

```diff
@@ -0,0 +1,38 @@
+package shorten
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+
+	"shorturl/internal/app/httpserver/constants"
+	httpmodels "shorturl/internal/app/httpserver/models"
+)
+
+func (r *shortenRouter) DeleteShortURLs(ctx *gin.Context) {
+	headerContentType := ctx.GetHeader("Content-Type")
+	isCorrectHeaderContentType := r.checkHeaderContentType(headerContentType)
+
+	if !isCorrectHeaderContentType {
```

Comment:
```suggestion
	if !r.checkHeaderContentType(headerContentType) {
```

точно нужна лишняя переменная?

Reply:
Это делает код более читаемым. Если кажется, что лишняя память берется под это, могу убрать